packages/react-data-grid/src/Cell.js

```diff
@@ -297,6 +297,18 @@ const Cell = React.createClass({
     }
   },
 
+  removeScroll() {
+    let ctrl: any = this; // flow on windows has an outdated react declaration, once that gets updated, we can remove this
+    if (ctrl.isMounted()) {
+      let node = ReactDOM.findDOMNode(this);
```

Comment:
We should add a ref on the container [element](https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/Cell.js#L502)

https://facebook.github.io/react/docs/refs-and-the-dom.html
"React will call the ref callback with the DOM element when the component mounts, and call it with null when it unmounts."
In theory, this should eliminate the need to check isMounted.